src/common/hotbar-store.ts

```diff
@@ -187,7 +187,7 @@ export class HotbarStore extends BaseStore<HotbarStoreModel> {
     } else if (0 <= cellIndex && cellIndex < hotbar.items.length) {
       hotbar.items[cellIndex] = newItem;
     } else {
-      logger.error(`[HOTBAR-STORE]: cannot pin entity to hotbar outside of index range`, { entityId: uid, hotbarId: hotbar.id, cellIndex, });
+      logger.error(`[HOTBAR-STORE]: cannot pin entity to hotbar outside of index range`, { entityId: uid, hotbarId: hotbar.id, cellIndex });
```

Comment:
I made the specification of requiring commas be `"always-multiline"`. This means that the commas are required on multiline objects (and arrays) and forbidden from single line ones.

The common reason to enforce dangling commas is to make diffs smaller when adding or removing entries. However, if the object or array is all on the same line anyway that benefit is moot.

Finally, I didn't think it would be a good idea to require commas in the `[ "entry" ]` example.

Reply:
I understand that the diffs benefit is moot but on the other hand the appearance difference is minimal if there's just one more comma for the single-line object. My point is now I have to remember when not to add the trailing comma instead of just thinking of the comma as an element terminator and always using it. Not a major point, I can handle it.  Just wondered if there was more of a reason than the moot diffs benefit.

Reply:
If you have

```
"editor.codeActionsOnSave": {
    "source.fixAll.eslint": true
},
```

turned on in vscode then you shouldn't have to remember anything.

Reply:
perfect, I just have to remember to do that.